.github/workflows/e2e-compatibility-workflow-call.yaml

```diff
@@ -40,6 +40,7 @@ jobs:
         uses: actions/checkout@v3
         with:
           repository: cosmos/ibc-go
+          ref: carlos/e2e-support-binary-new-genesis-commands
```

Comment:
we shouldn't need to change this at all 🤔 

Just realising now, you are just temporarily using this to run icad tests right? You can do this in an easier way via

`Actions` > `Manual E2E (Icad)` > 
<img width="255" alt="image" src="https://user-images.githubusercontent.com/7786534/226409856-e04de72e-1907-4b43-8b8b-62822d38bb86.png">

Reply:
> Just realising now, you are just temporarily using this to run icad tests right? 

Yes, that's right. And I needed this to be triggered from the interchain-accounts-demo repository.